packages/preferences/src/browser/style/search-input.css

```diff
@@ -15,30 +15,54 @@
  ********************************************************************************/
 
 .theia-settings-container .settings-search-container {
-  width: 100%;
-  position: relative;
+  display: flex;
+  align-items: center;
 }
 
 .theia-settings-container .settings-search-container .settings-search-input {
-  width: 100%;
+  flex: 1;
   text-indent: 8px;
   padding: calc(var(--theia-ui-padding) / 2) 0;
   box-sizing: border-box;
   border: 1px solid var(--theia-dropdown-border);
 }
 
-.theia-settings-container .settings-search-container .settings-search-icon {
+.theia-settings-container .settings-search-container .option-buttons {
+  height: 23px;
+  align-items: center;
   position: absolute;
-  height: 100%;
-  left: 8px;
-  font-size: var(--theia-ui-font-size0);
-  display: -webkit-box;
-  display: -ms-flexbox;
+  z-index: 999;
+  right: 5px;
   display: flex;
-  -webkit-box-pack: center;
-  -ms-flex-pack: center;
-  justify-content: center;
-  -webkit-box-align: center;
-  -ms-flex-align: center;
-  align-items: center;
+}
+
+.theia-settings-container .settings-search-container .clear-all {
+  background: var(--theia-icon-clear);
+}
+
+.theia-settings-container .settings-search-container .results-found {
+  background-color: var(--theia-badge-background);
+  border-radius: 2px;
+  color: var(--theia-badge-foreground);
+  font-size: smaller;
+  opacity: 0.7;
```

Comment:
I believe these two properties can be dropped, without them the badge is clearer and easier to read for end-users:

![badge](https://user-images.githubusercontent.com/40359487/104323711-7e047580-54b4-11eb-9777-f7cc3edeacba.png)

What do you think?